pages/common/gh-browse.md

```diff
@@ -0,0 +1,36 @@
+# gh browse
+
+> Open a GitHub repository in the browser or print the URL.
+> More information: <https://cli.github.com/manual/gh_browse>.
+
+- Open the home page of the current repository in the default web browser:
+
+`gh browse`
+
+- Open the home page of a specific repository in the default web browser:
+
+`gh browse {{owner}}/{{repository}}`
+
+- Open the settings page of the current repository in the default web browser:
+
+`gh browse --settings`
+
+- Open the wiki of the current repository in the default web browser:
+
+`gh browse --wiki`
+
+- Open a specific issue or pull request in the web browser:
+
+`gh browse {{issue_or_pull_request_number}}`
+
+- Open a specific branch in the web browser:
+
+`gh browse --branch {{branch_name}}`
+
+- Open a specific file or directory of the current repository in the web browser (the path should be relative to the repository):
+
+`gh browse {{path/to/file_or_folder}}`
+
+- Display the destination URL without open the web browser:
```

Comment:
```suggestion
- Print the destination URL without open the web browser:
```

This may be like my newcoming of disliking display but print works better here potentially since it was used in the title description